README.md

```diff
@@ -101,6 +97,35 @@ func main() {
 }
 ```
 
+In order to Marshal/Unamrshal Ion values with annotation, we use a Go struct of type []stirng and tag it with
+`ion:",annotation"`. The other field would be what our Ion value can fit in (both in terms of type and size). For instance,
+to Marshal `age::20`, it must be in a struct as below:
+```GO
+  type foo struct {
+    Value   interface{}
+    AnyName []string `ion:",annotations"`
+  }
+  data := foo{20, []string{"age"}}
+  val, err := MarshalText(data)
+  if err != nil {
+     t.Fatal(err)
```

Comment:
The goal is to have, as close as possible, copy paste code that works. `t.Fatal` seems like a leftover from a copy paste from our tests :)

Reply:
Ah! There is always one which gets away. Thanks for noticing it.